### DISCUSSION TITLE
Docker image with notebooks

### DISCUSSION THREAD

**Discussion by artyomtugaryov** [openvinotoolkit/openvino_notebooks#12] [Ideas]
Hello!

I just want to suggest building docker images with these notebooks. I think it will be useful and convenient to use these notebooks in dockers.

**Comment by ryanloney** [comment]
I started to experiment with building a Docker image using UBI for the Open Data Hub. It is only for testing purposes, but we plan to offer something like this soon. Let us know if you have any suggestions/advice. 

Dockerfile: https://github.com/ryanloney/openvino_notebooks/blob/odh/Dockerfile

Test image (not meant for distribution)
`quay.io/ryanloney/openvino_notebooks:latest`

If you forward port 8888 when running the image, it should launch Jupyter Lab in the openvino_notebooks directory.
